Scripts/Source/minai_Sex.psc

```diff
@@ -194,84 +194,97 @@ Function StartSexOrSwitchToGroup(actor[] actors, actor akSpeaker, string tags=""
   Main.Info("Sex: Starting/switching for " + actors.Length + " actors (tags: " + tags + ")")
   bool bSpeakerInScene = False
   bool bPlayerInScene = False
-  int i =0
-  int numMales = 0
-  int numFemales = 0
   bool bCanAnimate = True
-  Actor[] actorsInScene
-  sslThreadController Controller
   bool isNewScene = False
   if bHasOstim && minai_UseOStim.GetValue() == 1.0
+    Actor[] ostimActors = new actor[5]
+    ostimActors = actors
     ; Ostim
+    ostimActors = actors
```

Comment:
Probably don't need to assign this twice in a row.

Reply:
I'm manipulating the array to add player in logic, so I wanted to leave a raw version for your code later.

Reply:
No, I mean, you're assigning the same variable two lines apart. :D 

line 201 and line 203 are the same.

Reply:
Oh, yes! I wanted to make really sure it got in there.